docs/source/api/to_extensions.rst

```diff
@@ -130,13 +132,13 @@ Request Structure
 
 	.. note:: This field has meaning only for "Check Extensions"
 
-:type:    The :term:`Type` of extension - there are a set number of allowed values which are not recorded anywhere at the time of this writing
+:type:    The :term:`Type` of extension. It must be a CHECK_EXTENSION type with the naming convention of ``CHECK_EXTENSION_*``.
```

Comment:
... and I think it's best to put things like that as a block-quote under the actual things that they reference. So:
```rst
:type: sample text

	.. versionchanged:: ATCv4 lorem ipsum dolor...

:version: text for your header
```